apps/emqx/src/config/emqx_config_logger.erl

```diff
@@ -150,30 +150,18 @@ tr_file_handler({HandlerName, SubConf}) ->
     {handler, atom(HandlerName), logger_disk_log_h, #{
         level => conf_get("level", SubConf),
         config => (log_handler_conf(SubConf))#{
-            type =>
-                case conf_get("rotation.enable", SubConf) of
-                    true -> wrap;
-                    _ -> halt
-                end,
-            file => conf_get("file", SubConf),
-            max_no_files => conf_get("rotation.count", SubConf),
-            max_no_bytes => conf_get("max_size", SubConf)
+            type => wrap,
```

Comment:
what will happen if the old configuration here is `halt`

Reply:
rotation(warp) is better for all situtation, we should  force to use `rotation`.